modules/configuring-a-proxy-during-installation-cli.adoc

```diff
@@ -6,7 +6,7 @@
 [id="configuring-a-proxy-during-installation-cli_{context}"]
 = Configuring a proxy during installation using the CLI
 
-If you are installing a {product-title} (ROSA) cluster into an existing Virtual Private Cloud (VPC), you can use the ROSA CLI (`rosa`) to enable a cluster-wide HTTP or HTTPS proxy during installation.
+If you are installing a {product-title} (ROSA) cluster into an existing Virtual Private Cloud (VPC), you can use the {product-title} (ROSA) CLI (`rosa`) to enable a cluster-wide HTTP or HTTPS proxy during installation.
```

Comment:
It doesn't make sense to use "{product-title} (ROSA)" twice in the same paragraph - the second instance should just be "ROSA"